### ISSUE TITLE
The 'name' value must be set in other when plugin disabled

### ISSUE THREAD

**Issue by leonstr** [PlagiarismCheck/moodle-plagiarism_pchkorg#27]
When saving changes to an activity when *Enable plugin* is "No" the error *Coding error detected, it must be fixed by a programmer: The 'name' value must be set in other.* appears.

With debugging enabled:

```
More information about this error
Debug info:
Error code: codingerror
Stack trace:

    line 136 of /lib/classes/event/course_module_updated.php: coding_exception thrown
    line 272 of /lib/classes/event/base.php: call to core\event\course_module_updated->validate_data()
    line 161 of /lib/classes/event/course_module_updated.php: call to core\event\base::create()
    line 695 of /course/modlib.php: call to core\event\course_module_updated::create_from_cm()
    line 168 of /course/modedit.php: call to update_moduleinfo()
```

Moodle 4.0.4+ (Build: 20220930), plagiarism_pchkorg v3.11.2 (2022092718).

Related to #25.

**Comment by leonstr** [comment]
Added PR #28 with proposed fix.

**Comment by JaneAdelmann** [comment]
PR has been merged. Fix in version v3.11.3.

Thanks you.
